### DISCUSSION TITLE
How does PrismLauncher's Microsoft Account Authentication Work

### DISCUSSION THREAD

**Discussion by yuckdevchan** [PrismLauncher/PrismLauncher#2700] [Q&A] [answered]
I love prism launcher and am currently attempting to make my own launcher program with flutter instead of Qt.

However one roadblock is MSA authentication to get the token to launch the game. I have no idea how it is implemented in PrismLauncher.

Do you use the MSMC auth library?
Does prism launcher pay for Azure services?
Do you have to use proxies?

**Comment by Trial97** [comment, marked as answer]
https://wiki.vg/Microsoft_Authentication_Scheme
Have fun reading
no MSMC auth lib
no we do not pay azure(the app key thing should be free)
no we do not use proxies

**Reply by yuckdevchan** [reply]
Thanks, I'll take a look at that. I couldn't find a useful page like that before, just some other pages on wiki.vg
